### DISCUSSION TITLE
renewable energy?

### DISCUSSION THREAD

**Discussion by afunuks** [community/community#55682] [Copilot Conversations]
### Select Topic Area

Question

### Body

I am about to sign-up for the program. But I want to understand what I am paying for.
Does it include the cost of computing and its energy footprint? Does it make sure that's produced by renewables like solar or hydro(which is less safe) or other sensible options?
Of course, it's going to pay salaries for the developers.
But for something so prevalent. At 10$ per month. It's usually not talked about its sustainability for corporate but for poor developers, we kind of want to check and compare options and it's not convincing on an ethic level which is not a strong strength of tech companies.

**Comment by Akash1134** [comment]
Hi @afunuks ,

First of all, thank you for asking such a great question. This is rare feedback and we love that you've shared it. 👍 

At GitHub, we believe in the extraordinary potential and power of a diverse, collaborative developer community to accelerate human progress. Our parent company, Microsoft, has made a [sweeping set of commitments](https://www.microsoft.com/en-us/corporate-responsibility/sustainability) to combat climate change across carbon, water, waste and ecosystems. Here's the [report](https://www.microsoft.com/en-us/corporate-responsibility/sustainability/report) from 2022.

In 2020, Microsoft announced that they will be a carbon negative, water positive, zero waste company that protects ecosystems—all by 2030. Three years into this journey, we continue to make steady progress and remain steadfast in our commitment.

I'd recommend that you read these blogs to understand what we are doing to create a sustainable future while we are excited and committed to take on new challenges.

- https://github.blog/2021-04-22-environmental-sustainability-github/ 
- https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RW14sJN
